bindings/matrix-sdk-crypto-ffi/src/verification.rs

```diff
@@ -23,15 +24,6 @@ pub trait SasListener: Send {
     fn on_change(&self, state: SasState);
 }
 
-impl<T: Fn(SasState)> SasListener for T
```

Comment:
This `impl` would make it possible to pass pure functions from the clients, but they actually implement their own `SasListener` protocol, so the default impl is not necessary